src/main/java/sender/Stream_BMS_Data.java

```diff
@@ -0,0 +1,43 @@
+package sender;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import com.google.gson.Gson;
+
+public class Stream_BMS_Data 
+{
+	ILogger logger;
+	List<BatteryParams> streamData = new ArrayList<>();
+
+	Stream_BMS_Data(ILogger logger)
+	{
+		this.logger = logger;
+	}
+	
+	public List<BatteryParams> streamData(int count) {
+		if(count <= 0)
+			return null;
+		
+		for (int i = 1; i <= count; i++) {
+			streamData.add(new BatteryParams()

```

Comment:
`BatteryParams` has only set and get methods. This indicates a missed opportunity for encapsulation.
E.g., consider including the conversion to JSON in the same class. Then, adding a new parameter will be more localized to that class. How much more related functionality can you encapsulate in it?